### ISSUE TITLE
Investigate table support / styling for blog posts

### ISSUE THREAD

**Issue by brdunfield** [cds-snc/digital-canada-ca#342]
Core markdown spec does not support tables, but most markdown renderers do.

Does Hugo?
If so, how does a table render in a blog post? Do we need CSS rules around it?

**Comment by khalidelaggan** [comment]
Hello @brdunfield  @lynnchalati  @victoriachanimal 

Do we need to look into table support for any future posts? has this topic been addressed before by anyone in the team?

I'm more than happy to assist with this feature if needed

Thanks
Khalid

**Comment by lynnchalati** [comment]
It was something we tried to do before but there wasn't a good way to do it and still be accessible and modular after talking to Julianna and Ross about it... or at least that's how I remember it. Do you have other suggestions?

**Comment by khalidelaggan** [comment]
Actually, a few things to address around this topic first.



I will break them down to a few questions and answers 

-  As a developer, does Core markdown spec support tables now? 
  Not sure yet, will have to do a quick research on this topic 
- As a website editor, is it easy to create a table in Markdown and publish it?
Possible, but not easy. The technique is unreliable and involves too much work involved using markdown
- As a developer, can we make markdown generated tables accessible?
Not possible at all
- As a developer, Is it possible to insert <HTML code> inside Markdown content in our template? Will it render properly? 
Yes, that's how we should approach this if needed. HTML is part of markdown because HTML-inside-markdown is part of the spec. The only caveat is that we will have to add all the CSS rules around it in our code base. Some parsers may not accept this approach. We can try it out and test the results in a fake page.
- As a website editor, does it mean that we will always need a dev resource to create these tables for us?
Yes 100% 
-  Can we make this HTML table that we inserted in markdown accessible?
Yes, because it's an HTML rendered table, it can be accessible for sure, there is always tips and tricks to get around the completion of this task.

Having said all of the above, if we haven't used tables before in our posts and we don't have a need for it at the moment, I will just recommend avoid using it entirely.

I hope that makes sense and I was able to provide a brief idea on this and happy to discuss further :)
